keras_dqn/agent.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+from keras.utils.np_utils import to_categorical
+
+
+class Agent(object):
+    def __init__(self, model, target_model=None, target_swap=None):
```

Comment:
do we want the agent to handle the model and target_model? Or should all those functions just be pushed into the network file?